keyboards/hadron/ver3/keymaps/sebaslayout/keymap.c

```diff
@@ -0,0 +1,71 @@
+#include QMK_KEYBOARD_H
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+	KEYMAP(
```

Comment:
We are trying to move away from using the `KEYMAP` macro name. Please change these to `LAYOUT`.